Product/SoapUI_Test/ValidationSuite/ConnectValidation-soapui-project.xml

```diff
@@ -48882,7 +48882,7 @@ sql.close()
 }</con:afterLoadScript>
 <con:wssContainer/>
 <con:databaseConnectionContainer/>
-<con:reporting>
+<con:oAuth2ProfileContainer/><con:reporting>
 	<con:xmlTemplates/>
```

Comment:
There should be no changes here, since you didn't update the ValidationSuite project.